Dockerfile.in

```diff
@@ -1,37 +1,49 @@
 FROM DISTRO
```

Comment:
Quick question because im curious, how this is working where does the dockerfile get the DISTRO image ?

Reply:
https://github.com/jellyfin/jellyfin-ffmpeg/blob/2f4e0116d8869fb3e2a7384e4bafff076e6eaeda/build#L105